lib/ui/prompt.js

```diff
@@ -76,9 +76,11 @@ PromptUI.prototype.processQuestion = function (question) {
 };
 
 PromptUI.prototype.fetchAnswer = function (question) {
+  var screen = this.createScreen();
   var Prompt = this.prompts[question.type];
-  var prompt = new Prompt(question, this.rl, this.answers);
+  var prompt = new Prompt(question, screen, this.rl, this.answers);
```

Comment:
This approach is way better in my opinion, but I think we can do something better:

Would it makes sense for the BaseUI to call a method (`forceClose` or `cleanup`) on the Prompt instance. Then the prompt release the cursor in screen manager. That approach would really mean we don't pass or reference the screen manager in objects that don't own it.